rpc/jsonrpc/types/methods.go

```diff
@@ -1266,6 +1266,21 @@ func NewWalletPassphraseChangeCmd(oldPassphrase, newPassphrase string) *WalletPa
 	}
 }
 
+// WalletPubPassphraseChangeCmd defines the walletpubpassphrasechange JSON-RPC command.
+type WalletPubPassphraseChangeCmd struct {
+	OldPassphrase string
+	NewPassphrase string
+}
+
+// NewWalletPubPassphraseChangeCmd returns a new instance which can be used to
+// issue a walletpubpassphrasechange JSON-RPC command.
+func NewWalletPubPassphraseChangeCmd(oldPassphrase, newPassphrase string) *WalletPubPassphraseChangeCmd {
```

Comment:
we're no longer adding these New* functions to this package.  In the next major break, all of them will be removed.  Create the parameters from the struct itself instead.

Reply:
The change has been made (the parameters have been created from the struct)...